src/pygeoprocessing/geoprocessing.py

```diff
@@ -1581,6 +1589,9 @@ def reproject_vector(
             reprojection step. If it is an iterable, it will contain the
             field names to exclusively copy. An unmatched fieldname will be
             ignored. If ``False`` no fields are copied into the new vector.
+        osr_axis_mapping_strategy (OSR object): The OSR Axis Mapping Strategy 
+            as the first element. Defaults to a GIS friendly Lon,Lat order 
+            defined at geoprocessing.DEFAULT_OSR_AXIS_MAPPING_STRATEGY.
```

Comment:
Could you change the docstring to the one I proposed above?

Reply:
Done in [2eba123](https://github.com/natcap/pygeoprocessing/pull/31/commits/2eba123c585913539377eeaa05ebccdf13990a80)